src/hooks/useCart/useCart.tsx

```diff
@@ -246,6 +246,8 @@ export const useCart = (
           e.message === "Invalid Purchase Request: Duplicate identifier inputs"
         ) {
           setCartError(new Error(ERROR_MESSAGE.DUPLICATE_IDENTIFIER_INPUT));
+        } else if (e.message === "Invalid Purchase Request: item not found") {
```

Comment:
Would this look better? 

```suggestion
        } else if (e.message === "Invalid Purchase Request: Item not found") {
```

If you change this, remember to change it in the backend as well!